LibraryManagement.Data/DataRequestObjects/Books/InsertBook.cs

```diff
@@ -0,0 +1,33 @@
+﻿using LibraryManagement.Data.Abstraction;
+
+namespace LibraryManagement.Data.DataRequestObjects.Books
+{
+    internal class InsertBook : IDataExecute
+    {
+        public string Title { get; set; }
+        public string PublishedYear { get; set; }
+        public string Summary { get; set; }
+
+        public InsertBook(string title, string publishedYear, string summary)
+        {
+            Title = title;
+            PublishedYear = publishedYear;
+            Summary = summary;
+        }
+
+        public object? GetParameters()
+        {
+            return new
```

Comment:
When the parameters are all the same name as the properties on the 'DataRequest' object, we can just 'return this' to return the class itself when the GetParameters() method is called.